### ISSUE TITLE
Feature request:  Right/Left advance/detract pages like most pdf readers

### ISSUE THREAD

**Issue by averyfreeman** [linuxmint/xreader#436]
```
 * Xreader version 2.6.4
 * Distribution - Fedora fc33.x86_64
```

This is not a bug report.  Feel free to ignore it, as I am sure you are busy.

**Issue**
Right/left arrow keys do not advance to next/prev page like behavior in most pdf viewers, requiring my most hated activity: reading of directions.

**Steps to reproduce**
1) Open pdf
2) Want to move to next page
3) Press right arrow key
4) notice nothing's happening
5) look up the docs for xreader
6) notice that it uses ctrl-pgdn/ctrl-pgup instead 
7) install evince in avoidance of accessing mental functions

**Expected behaviour**
Right/Left to advance/detract page

**Other information**
This is a feature request.  I'd love to use x Reader, as I love pretty much everything about Cinnamon, but I cannot abide such blatant disregard of non-explicit, unplanned, defacto standardization.  

Cinnamon is greatly appreciated by us mindless creatures of habit because it's basically just like a better version of Windows.  I try what I need and it responds how I expect, and that makes me feel warm and fuzzy inside.

Like hot coco on a cold winter morning, chicken soup when you're sick, or computers looking like Mac and Windows.

Therefore, I implore you modify xReader to behave more closely to Adobe Reader, otherwise you risk confusing people like me with short attention spans and long muscle memory. 

Other than that, keep up the good work!

**Comment by scytho** [comment]
+1!

(Oh Github, remove that device verification …)

**Comment by wujastyk** [comment]
+1

**Comment by scytho** [comment]
I am afraid we may as well wish for us to grow telekinetic powers, or so …
https://github.com/linuxmint/xreader/issues/466

**Comment by rgreen5** [comment]
+1 to the request. Or at least an easy way to change shortcut preferences.

**Comment by scytho** [comment]
I have a feeling that there is [no actual reason](https://github.com/linuxmint/xreader/issues/466#issuecomment-787609093) for such a thing.

**Comment by wujastyk** [comment]
@scytho  yes, I did.  When the page is bigger than fitting in a window, key behaviour changes (or appears to change).

I often find that my attempts to page through a document end up rotating the pages on the screen, because my years-old muscle-memory was trained on other programs.

**Comment by tcubano** [comment]
Up! Yes, this drives me mad. It's always strange whenever I need to use xreader and the left/right arrows don't work as habit from all the other pdf readers I've ever used. ctrl+pg up/down is not a viable solution.

**Comment by jerkey** [comment]
it's really telling that there has been no progress on this issue, just searching for "key" in issues shows that many people have come to point out the obvious.  I honestly wonder what's going on with the maintainers

**Comment by SkullHex2** [comment]
> it's really telling that there has been no progress on this issue, just searching for "key" in issues shows that many people have come to point out the obvious. I honestly wonder what's going on with the maintainers

Completely normal. They wait for the issue to go stale, hoping in the meantime that an update has (slightly) changed the steps to reproduce. Then they'll close the issue (even if it's still there, but the steps to reproduce don't apply anymore, so 🤷‍♂️), and if you try to say anything they'll lock it to contributors only.
